issue_processing/jira_issue_processing.py

```diff
@@ -138,11 +173,12 @@ def merge_user_with_user_from_csv(user, persons):
     return new_user
 
 
-def parse_xml(issue_data, persons):
+def parse_xml(issue_data, persons, skip_history):
     """Parse issues from the xml-data
```

Comment:
```
"""
Parse issues from the xml-data
```
Maybe, you can add a line break here to get consistent comments. Thank you, @fehnkera !